packages/neuron-ui/src/components/BroadcastTransaction/index.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useCallback, useMemo, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useNavigate } from 'react-router-dom'
+import { isSuccessResponse, RoutePath, isMainnet as isMainnetUtil, useDidMount, useGoBack, getExplorerUrl } from 'utils'
+import Dialog from 'widgets/Dialog'
+import AlertDialog from 'widgets/AlertDialog'
+import { useDispatch, useState as useGlobalState } from 'states'
+import {
+  broadcastTransactionOnly,
+  getCurrentWallet,
+  OfflineSignStatus,
+  openExternal,
+  getLiveCells,
+} from 'services/remote'
+import { ReactComponent as HardWalletIcon } from 'widgets/Icons/HardWallet.svg'
+
+import styles from './broadcastTransaction.module.scss'
+
+const BroadcastTransaction = () => {
+  const {
+    app: { loadedTransaction = {} },
+    chain: { networkID },
+    settings: { networks },
+  } = useGlobalState()
+
+  const [wallet, setWallet] = useState<State.Wallet | null>(null)
```

Comment:
No wallet should be included if this component is only for broadcasting a transaction unless it should be redirected to the history when the broadcasted transaction includes addresses of the current wallet, but I didn't see this logic in this component.

Reply:
The wallet name needs to be displayed at the top.

![image](https://github.com/nervosnetwork/neuron/assets/11756869/47e35c5a-5819-4e80-9698-bd5ed2a37520)

Reply:
How about we remove the 'Wallet' field?

Reply:
> How about we remove the 'Wallet' field?

cc @Keith-CY

Reply:
> > How about we remove the 'Wallet' field?
> 
> cc @Keith-CY

I will consider the `wallet` field useless in the dialog because there's no strong relationship between the transaction to broadcast and the active wallet. So, removing the wallet field looks good to me.